packages/expo-screen-capture/android/src/main/kotlin/expo/modules/screencapture/ScreenCaptureModule.kt

```diff
@@ -0,0 +1,68 @@
+package expo.modules.screencapture
+
+import android.app.Activity
+import android.content.Context
+import android.view.WindowManager
+
+import org.unimodules.core.ExportedModule
+import org.unimodules.core.ModuleRegistry
+import org.unimodules.core.errors.CurrentActivityNotFoundException
+import org.unimodules.core.Promise
+import org.unimodules.core.interfaces.ExpoMethod
+import org.unimodules.core.interfaces.ActivityProvider
+
+class ScreenCaptureModule(context: Context) : ExportedModule(context) {
+
+  private lateinit var activityProvider: ActivityProvider
+
+  override fun getName(): String {
+    return NAME
+  }
+
+  override fun onCreate(moduleRegistry: ModuleRegistry) {
+    activityProvider = moduleRegistry.getModule(ActivityProvider::class.java)
+  }
+
+  @ExpoMethod
+  fun activatePreventScreenCapture(promise: Promise) {
```

Comment:
Might be clearer just to name these methods `preventScreenCapture` and `allowScreenCapture` or `activateScreenCapturePrevention`.